modules/lkqdBidAdapter.js

```diff
@@ -182,29 +196,34 @@ function interpretResponse(serverResponse, bidRequest) {
     } else {
       try {
         let bidResponse = {};
-        if (bidRequest && bidRequest.data && bidRequest.data.bidId && bidRequest.data.bidId !== '') {
-          let sspXmlString = serverResponse.body;
-          let sspXml = new window.DOMParser().parseFromString(sspXmlString, 'text/xml');
-          if (sspXml && sspXml.getElementsByTagName('parsererror').length == 0) {
-            let sspUrl = bidRequest.url.concat();
-
-            bidResponse.requestId = bidRequest.data.bidId;
-            bidResponse.bidderCode = BIDDER_CODE;
-            bidResponse.ad = '';
-            bidResponse.cpm = parseFloat(sspXml.getElementsByTagName('Pricing')[0].textContent);
-            bidResponse.width = bidRequest.data.bidWidth;
-            bidResponse.height = bidRequest.data.bidHeight;
-            bidResponse.ttl = BID_TTL_DEFAULT;
-            bidResponse.creativeId = sspXml.getElementsByTagName('Ad')[0].getAttribute('id');
-            bidResponse.currency = sspXml.getElementsByTagName('Pricing')[0].getAttribute('currency');
-            bidResponse.netRevenue = true;
-            bidResponse.vastUrl = sspUrl;
-            bidResponse.vastXml = sspXmlString;
-            bidResponse.mediaType = VIDEO;
-
-            bidResponses.push(bidResponse);
+        if (bidRequest && bidRequest.data && typeof bidRequest.data === 'string') {
+          let sspData = new URLSearchParams(bidRequest.data);
```

Comment:
`URLSearchParams` is not supported in IE. See [docs](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)

Reply:
Fixed